usr/bin/ranger-usersync

```diff
@@ -25,7 +27,7 @@ action=`echo $action | tr '[:lower:]' '[:upper:]'`
 realScriptPath=`readlink -f $0`
 realScriptDir=`dirname $realScriptPath`
 cd $realScriptDir
-cdir=`pwd`
+cdir=/etc/ranger/usersync
```

Comment:
If possible, I prefer setting an env variable for /etc/ranger. Hard-coding might be a problem for the future releases.